sdk/core/Azure.Core/perf/DownloadTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.Test.Perf;
+
+namespace Azure.Template.Perf
+{
+    public class DownloadTest : PerfTest<DownloadTestOptions>
+    {
+        // Stream.CopyToAsync default buffer
+        private const int BufferSize = 81920;
+        private static InProcTestServer _server;
+        private static HttpPipeline _pipeline;
```

Comment:
Per chat, I believe we agreed to keep `HttpPipeline` per-instance.

Reply:
We've chatted more. I decided to keep tests as similar as possible and considering we recommend customers to keep clients as singletons having a static `HttpPipeline` is reasonable.